src/HomeComponent/HomeComponent.js

```diff
@@ -19,28 +20,31 @@ class HomeComponent extends Component {
   handleClick = () => {
     this.setState({ gif: this.state.list[Math.floor(Math.random() * this.state.list.length)]});
   }
-  
+
   componentDidMount() {
-    console.log('componentDidMount')
-    axios.get(`https://api.giphy.com/v1/gifs/search?api_key=${apikey}&q=cheeseburgers`).then((response) => {
+    // console.log('componentDidMount')
+    axios.get(`https://api.giphy.com/v1/gifs/search?api_key=${apikey}&q=cats`).then((response) => {
       console.log('response', response)
       const { data } = response.data;
       this.setState({ gif: data[Math.floor(Math.random() * data.length)], list: data });
     })
   }
-  // componentDidUpdate() {
-  //   console.log('componentDidUpdate')
-  // }
-  // componentWillUpdate() {
-  //   console.log('componentWillUpdate')
-  // }
-  // componentWillReceiveProps() {
-  //   console.log('componentWillReceiveProps')
-  // }
+  
   handleSearchBar = (input)=>{
```

Comment:
Good! Looks like you got it. Since you are capturing the user's input and using that to make a request, we actually don't need `newInput` anymore in our state. If you notice, we set newInput, but we don't ever use it anywhere in our code. Therefore, you can remove newInput from `state` and remove the `this.setState({ newInput: input })` code from here.